plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/AbstractTestIcebergConnectorTest.java

```diff
@@ -203,6 +209,46 @@ public void testShowCreateTable()
                         ")");
     }
 
+    @Override
+    protected void checkInformationSchemaTablesForPointedQueryForMaterializedView(String schemaName, String viewName)
+    {
+        // TODO The query should not fail, obviously. It should return the viewName, as information_schema.tables returns it when invoked without filters
```

Comment:
issue link

Reply:
@raunaqmorarka is there an issue for fix Iceberg MV reporting?

Reply:
BTW Do we have any roadmap issue for materialized views?